content-src/styles/_mixins.scss

```diff
@@ -15,3 +15,9 @@
   max-height: 1em * $line-count * $line-height / $font-size;
   overflow: hidden;
 }
+
+@mixin dark-theme-only {
+  [lwt-newtab-brighttext]:not(.force-light-theme) & {
+    @content;
+  }
+}
```

Comment:
Can you explain what's going on here?

Reply:
Ah I think I get it, nvm.

Reply:
Sure, it's a mixin that can wrap CSS declarations to make them only apply to the selector they're nested under in dark mode.

Reply:
It basically allows dark mode code to be declared as overrides adjacent to the light mode defaults.